cpp/src/parquet/column_writer.cc

```diff
@@ -2335,28 +2466,28 @@ std::shared_ptr<ColumnWriter> ColumnWriter::Make(ColumnChunkMetaDataBuilder* met
   switch (descr->physical_type()) {
     case Type::BOOLEAN:
       return std::make_shared<TypedColumnWriterImpl<BooleanType>>(
-          metadata, std::move(pager), use_dictionary, encoding, properties);
+          metadata, std::move(pager), use_dictionary, encoding, properties, bloom_filter);
```

Comment:
Shouldn't we raise an error if `bloom_filter` is not null here?

Reply:
Nice catch!